pynicotine/gtkgui/transfers.py

```diff
@@ -102,7 +102,7 @@ class Transfers:
     UNKNOWN_STATUS_PRIORITY = 1000
 
     path_separator = path_label = retry_label = abort_label = None
-    transfer_page = user_counter = file_counter = expand_button = expand_icon = grouping_button = None
+    transfer_page = user_counter = file_counter = expand_button = expand_icon = grouping_button = status_label = None
```

Comment:
If we have a reference to the status_label widget in this module, then `set_connection_stats()` can be moved in here too.